fastlane/lib/fastlane/actions/update_project_team.rb

```diff
@@ -5,18 +5,20 @@ module SharedValues
 
     class UpdateProjectTeamAction < Action
       def self.run(params)
-        path = params[:path]
+        project_path = params[:path]
+        target_name = params[:target]
 
-        # If a target is passed in then update it using the xcodeproj, else use simple regex
-        if params[:target]
-          UI.user_error!("Could not find path to xcodeproj '#{path}'. Pass the path to your project (not workspace)!") unless File.exist?(project_path)
-          target_name = params[:target]
+        UI.user_error!("Could not find path to xcodeproj '#{project_path}'. Pass the path to your project (not workspace)!") unless File.exist?(project_path)
 
-          # Load .xcodeproj
-          project = Xcodeproj::Project.open(project_path)
+        # Load .xcodeproj
+        project = Xcodeproj::Project.open(project_path)
 
-          # Fetch target
-          target = project.native_targets.find { |native_target| native_target.name == target_name }
+        # Fetch target
+        targets = project.native_targets
+        if params[:target]
```

Comment:
Since we are defaulting to all targets, maybe we should just turn this into an array `targets`?

Reply:
Good call good call! I like that idea... I will make that change quick 😉